app/src/main/java/com/example/giphyservice/data/repository/GifsRepositoryResult.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.giphyservice.data.repository
+
+import com.example.giphyservice.data.model.DataResult
+
+sealed class GifsRepositoryResult {
+    data class Success(val dataResult: DataResult) : GifsRepositoryResult()
```

Comment:
that class looks like a generic one. What do you think about renaming it to something like "RepositoryResul" and making it generic to mention which data type we can return in case of success